### PR TITLE
Remove references to FMW

### PR DESCRIPTION
**Pull request by rosemarymarano**
Including all references to JRF and JRF-restricted domains, as per [WKO - Documentation Remove FMW Infra support in Model in Image pattern](https://jira.oraclecorp.com/jira/browse/OWLS-106404?filter=-1). This first pass is in the 4.0 docs; it will be duplicated in the 3.4 docs after incorporating technical review comments.

Acknowledged that these references must **also** be removed from the MII sample and sample docs.

### CODE REVIEW

File: documentation/4.0/content/managing-domains/model-in-image/usage.md [hunk 1]
```diff
@@ -204,106 +202,3 @@ For more information see:
 - [Using a JDBC Store](https://www.oracle.com/pls/topic/lookup?ctx=en/middleware/fusion-middleware/weblogic-server/12.2.1.4/perfm&id=STORE-GUID-328FD4C1-91C4-4901-B1C4-97B9D93B2ECE) in _Administering the WebLogic Persistent Store_.
 - [High Availability Best Practices](https://docs.oracle.com/en/middleware/fusion-middleware/weblogic-server/12.2.1.4/jmsad/best_practice.html#GUID-FB97F9A2-E6FA-4C51-B74E-A2A5DDB43B8C) in _Administering JMS Resources for Oracle WebLogic Server_.
 - [Leasing](https://docs.oracle.com/pls/topic/lookup?ctx=en/middleware/fusion-middleware/weblogic-server/12.2.1.4/jmsad&id=CLUST-GUID-8F7348E2-7C45-4A6B-A72C-D1FB51A8E83F) in _Administering Clusters for Oracle WebLogic Server_.
-
-### Requirements for JRF domain types
```
Comment (mriccell) [inline documentation/4.0/content/managing-domains/model-in-image/usage.md:208]: There is a lot of good information in this section and I  do not want to lose it. We might want to remove MII references and move the section as its oun page after Domain and Cluster Resources?  I am open to better suggestions.  @robertpatrick thoughts?

**Review by mriccell — CHANGES_REQUESTED** [review]

File: documentation/4.0/content/managing-domains/model-in-image/usage.md [hunk 2]
```diff
@@ -172,7 +172,7 @@ The following Domain fields are specific to Model in Image domains.
 | `configuration.model.configMap`              | Optional. Set if you have stored additional models in a ConfigMap as per [Optional WDT model ConfigMap](#optional-wdt-model-configmap). |
 | `configuration.secrets`                      | Optional. Set this array if your image or ConfigMap models contain macros that reference custom Kubernetes Secrets. For example, if your macros depend on secrets `my-secret` and `my-other-secret`, then set to `[my-secret, my-other-secret]`.|
 | `configuration.model.runtimeEncryptionSecret`| Required. All Model in Image domains must specify a runtime encryption secret. See [Required runtime encryption secret](#required-runtime-encryption-secret). |
-| `configuration.model.domainType`             | Set the type of domain. Valid values are `WLS`, `JRF`, and `RestrictedJRF`, where `WLS` is the default. See [WDT Domain Types](https://oracle.github.io/weblogic-deploy-tooling/userguide/tools-config/domain_def/).|
+| `configuration.model.domainType`             | Set the type of domain. `WLS` is the default. See [WDT Domain Types](https://oracle.github.io/weblogic-deploy-tooling/userguide/tools-config/domain_def/).|
```
Comment (tbarnes-us) [inline documentation/4.0/content/managing-domains/model-in-image/usage.md:175]: @rjeberhard I assume this is deprecated and the advice should be something like "Do not set `domainType` to any other value than `WLS`."?

File: documentation/4.0/content/managing-domains/model-in-image/usage.md [hunk 3]
```diff
@@ -187,11 +187,9 @@ that are copied from [Auxiliary Images]({{<relref "/managing-domains/model-in-im
...
 
- - There are also additional fields that are specific to JRF domain types. For more information, see [Requirements for JRF domain types](#requirements-for-jrf-domain-types).
-
  - For fully specified Model in Image Domain YAML file examples,
    see the [`kubernetes/samples/scripts/create-weblogic-domain/model-in-image/domain-resources`](https://github.com/oracle/weblogic-kubernetes-operator/tree/{{< latestMinorVersion >}}/kubernetes/samples/scripts/create-weblogic-domain/model-in-image/domain-resources) GitHub directory for the [Model in Image sample]({{< relref "/samples/domains/model-in-image/_index.md" >}}).
-   The `WLS` and `JRF` subdirectories in this directory correspond to the `configuration.model.domainType`.
+   The `WLS` subdirectory in this directory corresponds to the `configuration.model.domainType`.
```
Comment (tbarnes-us) [inline documentation/4.0/content/managing-domains/model-in-image/usage.md:192]: @rjeberhard @rosemarymarano Will we be removing the JRF related directories? If so, then this sentence can be removed.

File: documentation/4.0/content/managing-domains/model-in-image/usage.md (hunk 1 above)
Comment (robertpatrick) [reply documentation/4.0/content/managing-domains/model-in-image/usage.md:208]: @mriccell As much as I wanted to find valuable content that might be usable in the future, I think all of this content has such a MII way of doing things bias that anything below would have to be completely rewritten for the Domain on PV way of doing things.

**Review by robertpatrick — APPROVED** [review]

**Comment by rjeberhard** [comment]
@mriccell @rosemarymarano, what's the verdict on this PR? How or if are we going to save this content other than source control history?

**Comment by rosemarymarano** [comment]
@rjeberhard No verdict thus far. Up to @mriccell

File: documentation/4.0/content/managing-domains/model-in-image/usage.md (hunk 1 above)
Comment (mriccell) [reply documentation/4.0/content/managing-domains/model-in-image/usage.md:208]: Understood then lets save the content for future docs when we add the enhancement to support it on PV

**Review by mriccell — APPROVED** [review]

File: documentation/4.0/content/managing-domains/model-in-image/usage.md (hunk 1 above)
Comment (robertpatrick) [reply documentation/4.0/content/managing-domains/model-in-image/usage.md:208]: ???  I just said that I do not believe there is any content worth saving and now you want to save it anyway?  The content is always retrievable from GitHub so there is really no action needed.

File: documentation/4.0/content/managing-domains/model-in-image/usage.md (hunk 3 above)
Comment (rjeberhard) [reply documentation/4.0/content/managing-domains/model-in-image/usage.md:192]: Tom is correct. Let's please remove this sentence.

File: documentation/4.0/content/managing-domains/model-in-image/usage.md (hunk 2 above)
Comment (robertpatrick) [reply documentation/4.0/content/managing-domains/model-in-image/usage.md:175]: @rjeberhard No.  I would be very much against leveraging anything in `.configuration.model` for this new functionality.

**Pull request merged**